src/components/full-calender/components/index.js

```diff
@@ -36,18 +36,15 @@ export default class EventCalender extends Component {
         this.props.fetchProrityEvents();
     }
 
-    componentWillReceiveProps(next){
-        console.log("cmp:::prev::",next);
-    }
-
     handlerCreateEvent = (e) => {
         const ObjId = Math.random().toString(36).substring(7);
 
-        this.props.CreateNewCalenderEvent({
-            label: e.current.value,
-            isSelected: true,
-            ObjId});
-
+        if (e.current.value.length !==0) {
```

Comment:
filed value need to be tested against regular expression
const isEmpty = value =>
    !(/^\s*$/.test(value)) && !(value === null);